doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,464 @@
+   "metadata": {
+    "solution2": "hidden"
+   },
+   "source": [
+    "```python\n",
+    "avg = np.average(bin_avgs)\n",
+    "sem = np.sqrt(np.sum((bin_avgs-avg)**2/((N_BINS-1.5)*N_BINS)))\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Best guess for measured quantity:\",avg)\n",
+    "print(\"Standard error of the mean:\",sem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we already have an estimate on how precise our simulation result is. But how do we know if we chose the appropriate bin size? The answer is, we can perform binning analysis for many different bin sizes and check when the SEM converges. For that we would like to define a function that does the binning analysis in one go."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "solution2": "hidden",
+    "solution2_first": true
+   },
+   "source": [
+    "#### Exercise\n",
+    "Define a function called ```do_binning_analysis``` that takes as arguments ```data``` (a numpy array containing the samples) and ```bin_size``` and returns the estimated SEM. You can reuse your code from the previous exercises and adapt it to be part of the function."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "solution2": "hidden"
+   },
+   "source": [
+    "```python\n",
+    "def do_binning_analysis(data, bin_size):\n",
+    "    n_samples = len(data)\n",
+    "    n_bins = n_samples // bin_size\n",
+    "    bin_avgs = np.zeros(n_bins)\n",
+    "    for i in range(n_bins):\n",
+    "        bin_avgs[i] = np.average(data[i*bin_size:(i+1)*bin_size])\n",
+    "    avg = np.average(bin_avgs)\n",
+    "    sem = np.sqrt(np.sum((bin_avgs-avg)**2)/((n_bins-1.5)*n_bins))\n",
+    "    return sem\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "solution2": "hidden",
+    "solution2_first": true
+   },
+   "source": [
+    "#### Exercise\n",
+    "Now take the data in ```time_series_1``` and perform binning analysis for bin sizes from 3 up to 5000 and plot the estimated SEMs against the bin size with logarithmic x axis. Your SEM estimates should be stored in a numpy array called ```sems```."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "solution2": "hidden"
+   },
+   "source": [
+    "```python\n",
+    "sizes = np.arange(3,5001,dtype=int)\n",
+    "sems = np.zeros(5001-3,dtype=float)\n",
+    "for s in range(len(sizes)):\n",
+    "    sems[s] = do_binning_analysis(time_series_1,sizes[s])\n",
+    "\n",
+    "plt.plot(sizes,sems, \"x\")\n",
+    "plt.xscale(\"log\")\n",
+    "plt.show()\n",
```

Comment:
By default, plots in Jupyter notebooks are quite small. We like to use a combination of `plt.rcParams.update({'font.size': 18})` and `fig = plt.figure(figsize=(10, 6))` to make them more readable. I've updated the [tutorial instructions](https://github.com/espressomd/espresso/wiki/Documentation#Tutorials) accordingly, it was an oversight on my part.

I've changed all plots accordingly and labeled all the axes: 67840aa3d9c19a776ff0363c4de96fbd9ceaa229